src/blas/backends/generic/CMakeLists.txt

```diff
@@ -17,11 +17,11 @@
 #
 #=========================================================================
 
-set(LIB_NAME onemath_blas_portblas)
+set(LIB_NAME onemath_blas_generic)
 set(LIB_OBJ ${LIB_NAME}_obj)
 
-if(NOT DEFINED PORTBLAS_TUNING_TARGET)
```

Comment:
Similarly to my comment, can we warn the user but keep things working if `PORTBLAS_TUNING_TARGET` is set?

Reply:
I added a warning message for both in  [e8d22aa](https://github.com/uxlfoundation/oneMath/pull/618/commits/e8d22aac5b8e6cd0d2d7e69aa0e1412eb193b79c)